content/2021-12-01-brigade-2-goes-ga.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Brigade 2 Goes GA"
+slug: "2021-12-01-brigade-2-goes-ga"
+description: "Brigade 2 is now generally available"
+date: "2021-12-01"
+author: "Kent Rancourt"
+summary: Brigade 2 is now generally available.
+---
+
+Alright, Brigadiers! This is the day we've all been waiting for. Brigade 2 is generally available! If you're ready to get started, head on over to our [QuickStart](https://docs.brigade.sh/intro/quickstart/).
+
+But that's not all!
+
+As promised in our previous post, we've given some overdue love to Brigade 1.x as it enters a six month maintenance window. Yesterday, we released v1.9.0 of our Helm chart. It installs the existing Brigade v1.4.0 software, but addresses the deprecation or removal or certain beta APIs from newer releases of Kubernetes, making v1.4.0 installable, for the first time, on the newest, shiniest clusters. Hot on its heels, we've also released Brigade v1.5.0, which primarily remediates high severity vulnerabilities in the worker's JavaScript dependencies. It's installable using Helm chart v1.10.0.
+
+Still, there's more!
+
+In an extraordinarily fortunate quirk of timing, CNCF is hosting our all-new, on-demand webinar starting _tomorrow_, December 2 -- coinciding almost perfectly with Brigade 2's GA release. It's available at your convenience, but only until December 8, so be sure to [RSVP soon](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-so-whats-new-in-brigade-2/).
```

Comment:
What's available? What becomes unavailable? Do you mean "You can start watching on Bevy as of Dec 2 and it will be available on youtube after that"?

Reply:
🤷‍♂️ I don't know the technical details. I just know that you have to RSVP with the link I included. Truth be told, it's already on YouTube, but I don't want to advertise that or even promise it will land there eventually until after Dec 8, otherwise it will cut into formal webinar attendance, which is what I really want.

Reply:
Okay, so maybe change "It's available at your convenience, but only until December 8," to "This webinar is available at your convenience through December 8," to sound less weirdly false-scarcity-having?

Reply:
Fixed in 907e114c629a5a6a340e71d9c3c65bd17f869dd8